purchase_order_secondary_unit/tests/test_purchase_order_secondary_unit.py

```diff
@@ -61,14 +61,18 @@ def test_purchase_order_01(self):
             self.assertEqual(line.product_qty, 7.0)
             # Test onchange product uom
             line.secondary_uom_qty = 3500.0
+            self.assertEqual(line.product_qty, 2450.0)
             line.product_uom = self.product_uom_gram
-            self.assertEqual(line.secondary_uom_qty, 3.5)
+            self.assertEqual(line.product_qty, 2450000.0)
+            self.assertEqual(line.secondary_uom_qty, 3500.0)
 
     def test_purchase_order_02(self):
         purchase_order = Form(self.order)
         with purchase_order.order_line.new() as line_new:
             # Test default purchase order line secondary uom
             line_new.product_id = self.product
+            self.assertAlmostEqual(line_new.product_qty, 0.7, places=2)
             self.assertEqual(line_new.secondary_uom_id, self.secondary_unit)
             self.assertEqual(line_new.secondary_uom_qty, 1.0)
-            self.assertAlmostEqual(line_new.product_qty, 0.7, places=2)
```

Comment:
If this reordering is not strictly needed, I would leave it as it was, so that in the migration we only have what is needed for the migration itself.

Reply:
Ok, it's my fault, I'll fix it

Reply:
Fixed

Reply:
Thanks